content/5-design-process.md

```diff
@@ -27,72 +27,88 @@ preview: process-preview.jpg
+**Direct conversations** with individual users can provide deeper insights into specific use cases and challenges. When conducting these conversations, be mindful of privacy concerns and use open source tools for collecting feedback. **Observations** of how people use the software in real situations can also reveal patterns and problems that users might not think to mention.
 
-**Usability testing** in open source can be both challenging and rewarding. While you might not have access to a traditional usability lab, you can leverage remote testing tools. Consider using platforms like UserTesting or UsabilityHub to reach a broader audience. Alternatively, you could organize community testing sessions, where volunteers from the project community participate in usability studies.
+Remember that contributing to user research is itself a valuable form of contribution to open source projects. Sharing your findings can help the entire community better understand user needs.
 
-## Gathering and Incorporating Feedback
+## Working with Feedback
 
-In open source projects, stakeholders come from diverse backgrounds and have varying levels of involvement with the project. You might be dealing with casual users, power users, contributors, maintainers, and even representatives from companies that rely on the software. Each of these groups may have different priorities and perspectives.
+Getting and using feedback effectively is crucial in open source projects. When requesting feedback, be specific about what you're looking for and set clear timeframes for responses. Consider the circumstances of your contributors – they may be volunteers working across different time zones.
 
-To effectively gather feedback, it's important to create multiple avenues for input. This could include:
+Make it easy for people to provide feedback asynchronously. This could include:
 
-1. **Design proposals**: Share your designs early and often. Use the project's preferred platforms (e.g., GitHub issues, wikis) to post design mockups and invite comments.
+1. **Design proposals**: Share your designs early and often. Use the project's preferred platforms to post design mockups and invite comments.
 
 2. **Community calls**: Many larger open source projects hold regular video calls. These can be excellent opportunities to present designs and gather real-time feedback.
 
 3. **Asynchronous discussions**: Set up threads in project forums or mailing lists for longer, more in-depth discussions about design directions.
 
 4. **Design sprints**: For major redesigns, consider organizing virtual design sprints where interested community members can participate in intensive, collaborative design sessions.
 
```

Comment:
5. Ask for feedback that is specific and actionable. Don't be afraid to ask specifically and clearly for what you need feedback on regarding design e.g. '"I need feedback if this design meets the needs of user X when they are doing Y action under Z circumstances". When you ask for any and all feedback be prepared for long, multi-threaded conversations of feedback on feedback.

Reply:
Thank you. I added this in [1b49a88](https://github.com/GBKS/opendesign.guide/pull/24/commits/1b49a885b2039de67ec3613b2e147a78d3ab6364).